### DISCUSSION TITLE
(Documentation) Trigger Webhook URL changes depending on "aggregator" toggle

### DISCUSSION THREAD

**Discussion by hanifaudah** [bahasa-ai/otomatis-guidance#16] [General]
This needs to be added to the documentation to inform developers that a trigger's webhook url will change if the aggregator toggle is changed :)

**Comment by mgilangjanuar** [comment]
Yup, we'll define the `aggregator` term more clear. Thank you!

**Comment by jundialwan** [comment]
Need write about type/variation of Webhook URL also @mgilangjanuar 

Aggregator (Otomatis-level) webhook URL, User-level webhook URL, etc
